Userland/Libraries/LibGfx/ImageFormats/JPEGXLCommon.h

```diff
@@ -66,6 +66,14 @@ ALWAYS_INLINE ErrorOr<u64> U64(LittleEndianInputBitStream& stream)
     return value;
 }
 
+ALWAYS_INLINE ErrorOr<f32> F16(LittleEndianInputBitStream& stream)
+{
+    u16 const bits16 = TRY(stream.read_bits(16));
+    auto const biased_exp = (bits16 >> 10) & 0x1F;
+    VERIFY(biased_exp != 31);
+    return bit_cast<_Float16>(bits16);
```

Comment:
Should we add a `using f16 = _Float16` to AK/Types.h?

Reply:
That would be nice, I will add it if we figure out how to fix the related oss-fuzz issue.